README.md

```diff
@@ -20,17 +19,7 @@ Use this action to convert issues into a unified JSON structure. Read the [Codel
     FAVORITE_DISH: ${{ steps.issue-parser.outputs.issueparser_favorite_dish }}
 ```
 
-## Migrate from v2 to v3
-
-```diff
--- uses: stefanbuck/github-issue-parser@v2
-+- uses: stefanbuck/github-issue-parser@v3
-  id: issue-parser
-  with:
-+   issue-body: ${{ github.event.issue.body }}
-    template-path: .github/ISSUE_TEMPLATE/bug-report.yml
-```
-
+The text to be parsed can be set explicitly using `issue-body` input, otherwise it can be left to use the default value of `${{ github.event.issue.body }}`.
```

Comment:
@lemeurherve @gr2m , what do we think about this line? 

I originally wrote it longer, but cut it down for brevity, and people don't really need to see the historical references here?

Originally wrote: 
> In version 3.0.0, new input for `issue-body` is added to mitigate script injection attacks (see: [#42](https://github.com/stefanbuck/github-issue-parser/pull/42)). Version 3.0.1 defaulted `issue-body` to `${{ github.event.issue.body }}` for convenience (see: [#47](https://github.com/stefanbuck/github-issue-parser/pull/47)). The text to be parsed can be set explicitly using `issue-body` input, otherwise it can be left to use the default value.

Reply:
I'd say it's great, we can iterate on it later, I'd get this PR merged